<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeIDWindow</class>
 <widget class="QMainWindow" name="ChangeIDWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改个人信息</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>270</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>原密码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>新密码(选填)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>添加WCAID(选填)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
